packages/features/bookings/Booker/components/OverlayCalendar/OverlayCalendarSettingsModal.tsx

```diff
@@ -0,0 +1,155 @@
+          <div className="no-scrollbar max-h-full overflow-y-scroll ">
+            {isLoading ? (
+              <SkeletonLoader />
+            ) : (
+              <>
+                {data?.connectedCalendars.length === 0 ? (
+                  <EmptyScreen
+                    Icon={Calendar}
+                    headline={t("no_calendar_installed")}
+                    description={t("no_calendar_installed_description")}
+                    buttonText={t("add_a_calendar")}
+                    buttonOnClick={() => router.push("/apps/categories/calendar")}
+                  />
+                ) : (
+                  <>
+                    {data?.connectedCalendars.map((item) => (
+                      <Fragment key={item.credentialId}>
+                        {item.error && !item.calendars && (
+                          <Alert severity="error" title={item.error.message} />
+                        )}
+                        {item?.error === undefined && item.calendars && (
+                          <ListItem className="flex-col rounded-md">
+                            <div className="flex w-full flex-1 items-center space-x-3 pb-4 rtl:space-x-reverse">
+                              {
+                                // eslint-disable-next-line @next/next/no-img-element
+                                item.integration.logo && (
+                                  <img
+                                    className={classNames(
+                                      "h-10 w-10",
+                                      item.integration.logo.includes("-dark") && "dark:invert"
+                                    )}
+                                    src={item.integration.logo}
+                                    alt={item.integration.title}
+                                  />
+                                )
+                              }
+                              <div className="flex-grow truncate pl-2">
+                                <ListItemTitle component="h3" className="space-x-2 rtl:space-x-reverse">
+                                  <Link href={`/apps/${item.integration.slug}`}>
+                                    {item.integration.name || item.integration.title}
+                                  </Link>
+                                </ListItemTitle>
+                                <ListItemText component="p">{item.primary.email}</ListItemText>
+                              </div>
+                            </div>
+                            <div className="border-subtle w-full border-t pt-4">
+                              <ul className="space-y-4">
+                                {item.calendars.map((cal, index) => {
+                                  const id = cal.integrationTitle ?? `calendar-switch-${index}`;
+                                  return (
+                                    <li className="flex gap-3" key={id}>
+                                      <Switch
```

Comment:
Currently, when enabling the overlay for the first time, no calendars are automatically selected. Should we, by default, select all calendars that are enabled in the app settings for preventing double bookings?

I don't think it is clear that the user has to click the settings button first to toggle a calendar

Reply:
I think we should tackle this in a follow up - let me know if you think this is a blocker

Reply:
No blocker, follow-up is fine 👍